File: Example/NKJMultiMovieCaptureViewDemo/ConfirmViewController.m
//
//  ConfirmViewController.m
//
//  Created by nakajijapan.
//  Copyright 2014 nakajijapan. All rights reserved.
//

#import "ConfirmViewController.h"

@interface ConfirmViewController ()
@property MPMoviePlayerController *movielayer;
@property AppDelegate *appDelegate;
@end

@implementation ConfirmViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];

    [self.view setBackgroundColor:[UIColor whiteColor]];
    
    self.appDelegate = [[UIApplication sharedApplication]delegate];
    NSLog(@"composedMoviePath = %@", self.appDelegate.composedMoviePath);
    
    if (!self.appDelegate.composedMoviePath) {
        NSLog(@"No, Composed File.");
        return;
    }
    
    NSURL* movieUrl = [NSURL fileURLWithPath:self.appDelegate.composedMoviePath];
    
    self.movielayer = [[MPMoviePlayerController alloc] initWithContentURL:movieUrl];
    self.movielayer.controlStyle = MPMovieControlStyleEmbedded;
    self.movielayer.scalingMode  = MPMovieScalingModeAspectFit;
    
    [self.movielayer setShouldAutoplay:NO];
    [[self.movielayer view] setBackgroundColor:[UIColor lightGrayColor]];
    [self.movielayer.view setFrame:CGRectMake(0.0f, 64.0f, 320, 320)];
    [self.movielayer prepareToPlay];
    
    [self.view addSubview:self.movielayer.view];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: NKJMultiMovieCaptureViewDemo/ConfirmViewController.swift
//
//  ConfirmViewController.swift
//  NKJMultiMovieCaptureView
//
//  Created by nakajijapan on 6/19/15.
//  Copyright (c) 2015 CocoaPods. All rights reserved.
//

import UIKit
import MediaPlayer

class ConfirmViewController: UIViewController, MPMediaPickerControllerDelegate {

    var moviePlayerController:MPMoviePlayerController!
    var appDelegate:AppDelegate!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        self.view.backgroundColor = UIColor.whiteColor()
        self.appDelegate = UIApplication.sharedApplication().delegate as! AppDelegate
        print("composedMoviePath = \(self.appDelegate.composedMoviePath)")
        
        if self.appDelegate.composedMoviePath == nil {
            print("No, Composed File.")
            return
        }
        
        let movieURL = NSURL(fileURLWithPath: self.appDelegate.composedMoviePath!)
        
        self.moviePlayerController = MPMoviePlayerController(contentURL: movieURL)
        self.moviePlayerController.controlStyle = MPMovieControlStyle.Embedded
        self.moviePlayerController.scalingMode = MPMovieScalingMode.AspectFit
        
        self.moviePlayerController.shouldAutoplay = false
        self.moviePlayerController.view.backgroundColor = UIColor.lightGrayColor()
        self.moviePlayerController.view.frame = CGRectMake(0, 64, CGRectGetWidth(self.view.bounds), CGRectGetWidth(self.view.bounds))
        self.moviePlayerController.prepareToPlay()
        
        self.view.addSubview(self.moviePlayerController.view)
    }

}
